app/models/support_day.rb

```diff
@@ -1,8 +1,23 @@
 class SupportDay < ApplicationRecord
   SUPPORT_TYPES = %w[dev ops ooh1 ooh2].freeze
+  MONTHS_IN_THE_PAST = 2
+  TOTAL_MONTHS = 12
```

Comment:
```suggestion
  MONTHS_IN_THE_PAST_TO_DISPLAY = 2
  TOTAL_MONTHS_TO_DISPLAY = 12
```

(if that's what these are)